docs/guides/admin/docs/configuration/database.md

```diff
@@ -54,7 +55,7 @@ Finally, set root user credentials by running
 ```
 
 
-### Creating a Database
+#### Creating a Database
```

Comment:
It's kind of confusing since you do this with PostgreSQL as well… kind of. I would just remove this heading.

```suggestion
```